active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -0,0 +1,618 @@
+# Window and Application API Spec
+
+This spec describes updates to the XAML Window and Application APIs to accommodate this move to support Desktop (win32) apps.
+
+## Spec Status
+In review. No scheduled yet for reviewing by the API Board. 
+
+
+## Execution Notes
+
+This API spec contains the the APIs under the following criteria:
+1. Already exists in Window.UI.Xaml.Window. 
+2. Are part of the Mininum Viable Product
+
+There are other proposed APIs proposed that will be part of the future phases. 
+> A Phase doesn't correspond with a product version, it's just a set of Sprints with allocated resources to execute this work.
+
+##   Design Agreement
+- The new Window implementation reuses the current Windows.UI.XAML.Window class and it will be placed under Microsoft.UI.XAML.Window. 
+- Current methods, events, and properties should still work for UWP and add the new Win32 support. 
+- These APIs use different code paths depending on whether WinUI 3 is running on Win32 or UWP. 
+- There are some APIs that don’t work on Win32 or UWP yet in this phase. Those APIs could return **null** or **throw** an exception depending on the case. 
+
+This agreement applies to the Application class too.
+
+## Window Class
+Namespace: Microsoft.UI.Xaml
+
+```CS
+public class Window : IClosable
+```
+
+### **Window management, appearance, and behaviors**
```

Comment:
>Window management, appearance, and behaviors [](start = 6, length = 44)

Please take a look at AppWindow and Forms for examples of basic knobs to customize the Window (frame style, show title bar, etc.):
[https://docs.microsoft.com/en-us/uwp/api/windows.ui.windowmanagement.appwindow](https://docs.microsoft.com/en-us/uwp/api/windows.ui.windowmanagement.appwindow)
[https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.form?view=netframework-4.8](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.form?view=netframework-4.8)